Rollbar/RollbarNotifier.m

```diff
@@ -430,16 +471,22 @@ - (void)queuePayload_OnlyCallOnThread:(NSDictionary *)payload {
     [[RollbarTelemetry sharedInstance] clearAllData];
 }
 
-- (BOOL)sendItems:(NSArray*)itemData withAccessToken:(NSString*)accessToken nextOffset:(NSUInteger)nextOffset {
+- (BOOL)sendItems:(NSArray*)itemData
+  withAccessToken:(NSString*)accessToken
+       nextOffset:(NSUInteger)nextOffset {
+    
     NSMutableArray *payloadItems = [NSMutableArray array];
```

Comment:
This section of code now makes an array, iterates through the input adding each item to the new array, then creates a dictionary with that new array at the data key, then calls truncate on the whole thing. We should either not create this new array or more likely we should be calling truncate inside the loop like we used to. This SDK is different than the others in that it sends batches of items, so the logic is a bit different here. We shouldn't be sending batches anymore but that is a task for a future issue. In the meantime, we should be calling truncate on each item rather than on the entire payload because I think that will end up doing what we want.